### PR TITLE
Improve glossary entry of automations

### PR DESCRIPTION
**Pull request by Hypfer**
## Proposed change

This PR attempts to better communicate what exactly automations are.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
Thanks, @Hypfer 👍 

Looking good!

File: source/_data/glossary.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - topic: Add-on
   description: "Add-ons provide additional, standalone, applications that can run beside Home Assistant. Most of these, add-on provided, applications can be integrated into Home Assistant using integrations. Examples of add-ons are: an MQTT broker, database service or a file server."
 - topic: Automation
-  description: "[Automations](/docs/automation/) offer the capability to call a service based on a simple or complex trigger. Automation allows a condition such as a sunset to cause an event, such as a light turning on."
+  description: "[Automations](/docs/automation/) connect one or more triggers to one or more actions in a "when trigger then do action" fashion with additional optional conditions. For example, an automation might connect the trigger "sunset" to the action "turn the lights on" but only if the condition "someone is home" is met. Pre-made automations for common use-cases are available via [the blueprints feature](/docs/automation/using_blueprints/).
```
Comment (frenck) [inline source/_data/glossary.yml]:
```suggestion
  description: "[Automations](/docs/automation/) connect one or more triggers to one or more actions in a 'when trigger then do action' fashion with additional optional conditions. For example, an automation might connect the trigger 'sunset' to the action 'turn the lights on' but only if the condition 'someone is home' is met. Pre-made automations for common use-cases are available via [the blueprints feature](/docs/automation/using_blueprints/).
```

**Comment by frenck** [comment]
Hmm something breaks this, pulling your branch to see what is going on

**Comment by Hypfer** [comment]
Whoops. There was a missing " at the end of the string

**Comment by frenck** [comment]
Ooh well spotted 👀 , didn't notice. Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @Hypfer 👍

**Pull request merged**